### DISCUSSION TITLE
Similar Projects

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#80] [General]
https://github.com/kahst/BirdNET/commit/5d882ecd275728c73293c5640d7c177b43386012#commitcomment-60909613 

[HaikuBox](https://haikubox.com/)

**Comment by mcguirepr89** [comment]
This looks great! https://sites.google.com/site/kenyabioacoustics/home
https://github.com/ciiram/BirdPy

**Comment by DougWelch** [comment]
Dear Patrick et al,

We have been active in southern Ontario, Canada since late 2018:
https://archive.org/details/hamiltonbioacousticsfieldrecordings?sort=-date
http://hamiltonnature.org/bioacoustics

Also in this set of recordings is monitoring from reserves in the 
Escarpment Biosphere Conservancy.

The database I have created is not yet online but sounds like it shares 
many overlaps with this effort. Tables include:
archive (metadata for the internet archive data)
detections (information for each BirdNET detection)
recorder (information about the source recorder for each recording)
sites (location information, including "What3Words" for each recorder site)
sounddirs (nested directory structure information based on a 
recorder/year-month structure)
soundfile (name of the soundfile, beginning and end times, ..., and site)
taxonomy (species codes and names)
weather (hourly weather conditions - from Environment Canada in these cases)

The species list is restricted to species ever recorded in our area and 
the intersection of that with the BirdNET model. eBird frequency is 
ignored since we turn up rare species and migrants that don't 
necessarily get reported on eBird.

At present, we have 1,699,755 detections with confidence 90% or greater.

Attached is a visualization of American Woodcock detections from a 
single site for 2021 (monitored continuously). Colored lines indicate 
morning nautical twilight, sunrise, solar noon, sunset, and evening 
nautical twilight.

I am happy to share more information with anyone interested.

Cheers,
Doug

**Comment by nilspupils** [comment]
Hi gang, 
just for an interesting read - these are cool projects, although not specifically birdsound related but citizen science, camera traps and raspberry-pi, all with great documentation: 

[SAFE Acoustics: An open-source, real-time eco-acoustic monitoring network in the tropical rainforests of Borneo](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13438)
by @sarabsethi

[Broad-scale applications of the Raspberry Pi: A review and guide for biologists](https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13652)
awesome writeup by @JolleJolles

[PICT: A low-cost, modular, open-source camera trap system to study plant–insect interactions](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13618)
Great Guide: 
[How to build and use "PICT"? A users-friendly practical guide](https://zenodo.org/record/4893325#.Yg-uz-jMLup)

[WiseEye: Next Generation Expandable and Programmable Camera Trap Platform for Wildlife Research (PDF)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5226779/pdf/pone.0169758.pdf)

Have a great weekend!

**Reply by mcguirepr89** [reply]
Very cool! Thanks, Nils! I want to build a PICT! But I also want to _build_ a PICT, like from scratch :)

**Reply by oliverk71** [reply]
Camera traps have always been disappointing to me. Especially regarding image quality. I once wanted to build a camera trigger to use my DSLR, but I did never finish it. A colleague told me later that there are ready-built triggers for sale. Also some cameras have built-in triggers, which are quite useful. For insects, a small sensor is nice and better suited than an DSLR or anything with a larger sensor, because of focus depth. My last smartphone was a Huawei p30 Pro (the last one with google apps) and it has a real good camera and most important: a dedicated macro camera. But I also used small lenses glued in front of a smartphone case for macros. However, I think, using a smartphone is an easy to use solution. Only thing is: You will need a camera app, which takes some shots as soon something moves in the picture and maybe allows manual focus.

**Reply by nilspupils** [reply]
@oliverk71: You are right. I am also disappointed with the image quality of camera traps (even professional ones). As you said, the image quality of smartphones has become so good (my wife has the p30 also!), it is quite puzzeling why the technology has not reached the trail/wildlife cameras. (I heard 99% percent come from one factory.) I hope the pi (zero) can be used as a platform when some issues are adressed, like reliable triggering, (auto) focus and power management (the zero will never become a battery powered device though).

**Reply by oliverk71** [reply]
It would be nice if smartphone cameras could be used, but that does not work. Pi can be used and also the new pi cam, which is not bad and offers the possibility to use a variety of lenses. However, I think for many cases a smartphone is a better choice. The p30 Pro for example is not that expensive any more. There are apps which make it useful, for example the app "open camera" offers manual focus control and much more. And there is another app called "trigger" which I did not test, yet, but it can trigger the camera. Also, smartphone's cameras do not need too much light and allow more focus depth (because of small sensor) than larger cameras. If the smartphone can take raw images, it's even better. 

Don't take me wrong, I like the pi, I like Arduino microcontrollers and sensors and I like making things. But it should at least be better suited than anything what can be bought for even a smaller amount of money. 

The trigger box I once planned should be able to use different sensors as triggers and it could be connected to a DSLR. Depending on what  animals you want to take images of, it would be nice. But: there are plenty of possibilities available for DSLR, including app / hardware combinations and I am pretty sure that some cameras have already built-in triggers. 

A former colleague of mine used to take images and movies from certain small mammals in Africa. She used Sony camcorders (with night vision and fullHD) and solar panels for her traps and usual wildlife camera traps. Not an easy job. 

In my last institute, we were studying insect-plant interactions, usually pollination, mostly of bees. Images of insect/plant interactions are nice, but we also did a lot of other stuff, including measuring power of insects and flowers and how certain bees fit to certain flowers (floral filters). We also made false colour photography to simulate bee view. Because bees see UV, but not red. So we took (reflected) UV images, blue and green and changed the channels of RGB to G-B-UV. I am now working in another institute, but I still make some images of flowers, but just for fun and usually UV induced flourescence.

**Reply by JolleJolles** [reply]
Thanks for the nice mention! Cheers

**Reply by nilspupils** [reply]
@JolleJolles: Really enjoyed your excellent papers! I hope you like BirdNET-Pi! 
Everyone else: Check this out: 
https://raspberrypi-guide.github.io/

**Comment by nilspupils** [comment]
https://oekofor.netlify.app/en/portfolio/ecopi-bird2d_en/

**Comment by nilspupils** [comment]
https://avesrares.wordpress.com/2022/02/27/nocmig-a-modern-approach/

**Comment by mattluttrell** [comment]
Some of my favorite other Pi projects I've written myself:

1) Wrote my own version of Ring doorbell using common GNU software like "motion"
2) Snapping turtle underwater webcam/temperature monitor
3) Home power monitoring using the UPS's USB monitoring port

Coming:
ADSB airplane transponder tracking

Did not write myself:
**Beehive monitoring that uses audio to determine hive health** / activity (very similar to this project)

**Reply by mcguirepr89** [reply]
I have been considering adding "motion" as a toggle feature that could snap photos when motion is detected around the BirdNET-Pi -- :thinking:

**Reply by Svardsten53** [reply]
Raspberry Pi's are fun. I'm also running an ADS-B flight radar from VRS, works perfect. Also digital clocks, a weather station and Home Assistant home automation.

![vrs](https://user-images.githubusercontent.com/62435779/161571825-cd8158ef-5a22-40de-81c6-ef9963faf2a1.png)

**Comment by nilspupils** [comment]
https://github.com/BirdVox/birdvoxdetect
by @lostanlen

**Comment by nilspupils** [comment]
https://github.com/mikkohei13/Loxpyt
by @mikkohei13

**Comment by nilspupils** [comment]
https://github.com/josafatburmeister/BirdSongIdentification
by @josafatburmeister

**Comment by nilspupils** [comment]
https://github.com/HaroldMills/Vesper
by @HaroldMills
Looks very interesting, don't really understand though...;)

**Comment by nilspupils** [comment]
[Design of an ultra-low-power energy-harvesting audio sensor for ecosystem monitoring](https://dial.uclouvain.be/memoire/ucl/en/object/thesis%3A25100)
Excellent thesis with lots of details for example using the AOM-5024 microphone and amplification.
By @MartinBraquet

**Comment by nilspupils** [comment]
[DIY binaural microphone](http://www.hifi-forum.de/viewthread-211-999.html)
Very detailed thread about building a binaural microphone. Lot of useful information here for microphone-builders. 
[German language]

**Comment by nilspupils** [comment]
https://github.com/umr-ds/BirdEdge
https://github.com/umr-ds/BirdEdge-Mic

by @umr-ds

They mention BirdNET-Pi in their publication!
